rconweb/api/views.py

```diff
@@ -71,27 +72,31 @@ def set_name(request):
     )
 
 
+# TODO: this is not an exposed endpoint
 @csrf_exempt
-@login_required(True)
+@login_required()
 @record_audit
 def set_temp_broadcast(request):
     return set_temp_msg(request, temporary_broadcast, "set_temp_broadcast")
 
 
+# TODO: this is not an exposed endpoint
 @csrf_exempt
-@login_required(True)
+@login_required()
 @record_audit
 def set_temp_welcome(request):
     return set_temp_msg(request, temporary_welcome, "set_temp_welcome")
 
 
 @csrf_exempt
+@permission_required("api.can_view_crcon_version", raise_exception=True)
```

Comment:
hmm this is called without requiring a login though

Reply:
Whoops, fixed!